ci-operator/step-registry/gather/installer-gather/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
```

Comment:
This is a very non-sno specific directory, the step-registry/gather people might want to OWN it as well

Reply:
The location is intentinal since the logic isn't specific to SNO.
I'll add the owners of  step-registry/gather to approvers